SagaLabs-Manager-Client/src/main/java/sagalabsmanagerclient/controllers/HomeController.java

```diff
@@ -159,4 +166,8 @@ private Button createTurnOffButton(String labName) {
     }
 
 
+    public void openCaptureLabWindow(String labName) {
+        ViewSwitcher.openCaptureLabWindow(labName);
```

Comment:
I don't particular like this. Har i forvejen brugt lang tid på at gøre, så viewswitcher ikke er static, men med listeners ift. switch af view

Reply:
Vil gerne havde din hjælp med at gøre det til en del af viewswitcher så. evt kan vi bare merge nu, da det hele egentlig virker, og så kan vi rette det i master.